keyboards/evolvbeta/keyboard.json

```
@@ -0,0 +1,287 @@
+{
+    "keyboard_name": "Evolv75 Beta",
+    "manufacturer": "NathanAlpha",
+    "url": "https://archetypemade.com/evolv-is-here/",
+    "maintainer": "Gondolindrim",
+    "usb": {
+        "vid": "0x7865",
+        "pid": "0x0E76",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "backlight": true,
+        "encoder": true,
+        "bootmagic": true
+    },
+    "matrix_pins": {
+        "cols": ["B12", "B14", "C6" , "C7" , "C8" , "C9", "A8", "A10", "A15", "C10", "B8", "C11", "C0", "C1", "C2", "A3"],
+        "rows": ["C4" , "A7", "A6" , "A5" , "C12", "B9" , "A4"] 
+    },
+    "diode_direction": "COL2ROW",
+    "eeprom": {
+        "wear_leveling": {
+            "backing_size": 4096
+        }
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A0", "pin_b": "A1", "resolution": 2},
+            {"pin_a": "C3", "pin_b": "A2", "resolution": 2}
+        ]
+    },
+    "ws2812": {
+        "driver": "pwm",
+        "pin": "B15"
+    },
+    "rgblight": {
+        "led_count": 30,
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "alternating": true,
+            "twinkle": true
+        },
+        "default": {
+            "animation": "rainbow_swirl",
+            "val": 200
+        }
+    },
+    "backlight": {
+      "pin": "B1",
+      "levels": 20
+    },
```

Backlight is being used solely for caps lock. Indicator light should be used here instead. 
```suggestion
    "indicators": {
      "caps_lock": "B1"
    },
```